pandas/tests/groupby/test_value_counts.py

```diff
@@ -79,3 +79,28 @@ def rebuild_index(df):
     # have to sort on index because of unstable sort on values
     left, right = map(rebuild_index, (left, right))  # xref GH9212
     tm.assert_series_equal(left.sort_index(), right.sort_index())
+
+
+@pytest.mark.parametrize("freq", ["1D", "2D", "1W", "1Y"])
+@pytest.mark.parametrize("size", [100, 1000])
+@pytest.mark.parametrize("frac", [0.1, 0.5, 1])
+def test_series_groupby_value_counts_with_grouper(freq, size, frac):
+    np.random.seed(42)
```

Comment:
Why the seed? Ideally this passes for any seed; if not that's a problem.

Reply:
For the reproducibility, I set the seed arbitrary value. Well, It's no problem to get rid of that line. And I just found several tests are also using `np.random.seed`: https://github.com/pandas-dev/pandas/search?q=np.random.seed&unscoped_q=np.random.seed

Reply:
Let's remove it.

Reply:
yes, no problem!